azure_arc_k8s_jumpstart/aks/arm_template/azuredeploy.parameters.json

```diff
@@ -21,7 +21,7 @@
             "value": "<Your Azure service principal password>"
         },
         "kubernetesVersion": {
-            "value": "<Kubernetes Version, e.g. 1.18.2>"
+            "value": "<Kubernetes Version, e.g. 1.23.5>"
         }
     }
 }
```

Comment:
Not shown in the comment, but I think we can drop the AKS prefix from the parameters file and set it as default in the deployment template. [See example}(https://github.com/microsoft/azure_arc/blob/main/azure_arc_data_jumpstart/aks/ARM/azuredeploy.json#L69)

Reply:
Added default value for dns prefix in main template.
Removed dnsprefix from parameters file and screenshot.